02-OOP-Inheritance/Ex05/src/2-05.js

```diff
@@ -0,0 +1,25 @@
+import {eventEmitter} from './classEventEmitter';
+import {Movie} from './classMovie';
+import {Actor} from './classActor';
+import {Logger} from './classLogger';
+
```

Comment:
That's the problem you didn't use the Logger. You've to set a new listener for the event `play`, specifically the listener will be Logger.log.

```
let MyLogger = new Logger();
let aMovie = new Movie('Title', 'year', 'actor');

aMovie.on('play', MyLogger.log); //Listener added
aMovie.play() //Once you execute the event play, all the callbacks associated will be executed.
```